react/src/components/molecules/EmergencyAlert/style.scss

```diff
@@ -0,0 +1,82 @@
+@import "global";
+@import '00-base/mixins';
+@import '02-molecules/emergency-alert';
+@import '02-molecules/emergency-header';
+
+
+.ma__emergency-alert--c-primary {
+  @include ma-button("normal","solid",$c-primary);
```

Comment:
maybe we should create a callout mixin instead of using the button one which includes hover style etc.